Builder/src/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(CodeBuilder STATIC "CodeBuilder.cpp")
```

Comment:
I would delete this file and move target_include_directories in the CMakeLists from Builder folder.

Reply:
Ok.

Reply:
Modified according to your indications.